src/App/Infrastructure/Ui/Assets/scss/app/_variables/_typography.scss

```diff
@@ -0,0 +1,36 @@
+// This file is part of the Php DDD Standard project.
+//
+// Copyright (c) 2017 LIN3S <info@lin3s.com>
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+//
+// @author Beñat Espiña <benatespina@gmail.com>
+
+$font-weight-light: 300;
+$font-weight-normal: 400;
+$font-weight-bold: 700;
+
+$font-family-open-sans: 'Open Sans', sans-serif;
+
+$font-size-36: 36px;
+$font-size-32: 32px;
+$font-size-28: 28px;
+$font-size-26: 26px;
```

Comment:
Leave 3 weights, the whole list does not make sense. Why you will need a list of available sizes if you have all of them?